tensorboard/plugins/beholder/README.md

```diff
@@ -0,0 +1,153 @@
+# The Beholder Plugin
+
+The Beholder plugin shows a live video feed of tensor data in TensorBoard during
+model training. It can display model variable values, arbitrary NumPy arrays
+(e.g. for gradients or activations), or pre-existing image frames.
+
+For now, Beholder should be considered a **beta release**.
+
+Beholder was originally developed by [Chris Anderson][chrisranderson], who also
+created this [demo video for Beholder][demo-video].
+
+[chrisranderson]: https://github.com/chrisranderson
+[demo-video]: https://www.youtube.com/watch?v=06HjEr0OX5k
+
+![The Beholder Dashboard](docs/beholder.png)
+
+## Instrumenting a Model
+
+To use Beholder, import and instantiate the `Beholder` class in your model code
+and from the training loop call the `update` method with a `Session` argument:
+
+```python
+from tensorboard.plugins.beholder import Beholder
```

Comment:
nit: Maybe

```python
from tensorboard.plugins import beholder as beholder_lib
...
beholder  = beholder_lib.Beholder(LOG_DIRECTORY)
...
```

Likewise with `BeholderHook` below.

Reply:
Hmm, are you referring to just changing the import style or also adding the literal `...` lines?

For the import, I debated about this since it's not google python style to import unscoped symbols (which is why I did it this way in the demo code) but I thought importing just Beholder would seem more natural and concise to the typical Python user.  But I can change it.  If I do, do you think I should just change the README or also the example code given in the inactive plugin dialog?

In the relatively near term I'd like to move all our "public" API endpoints under `import tensorboard as tb` anyway, so this would presumably then become `import tensorboard as tb` followed by the (admittedly lengthy) `beholder = tb.plugins.beholder.Beholder(LOG_DIRECTORY)`.

Reply:
I noticed the code itself is changed in other files and was wondering if we wanted to make this match, but that plan makes sense. We can keep it like this.